graph/schema.graphqls

```diff
@@ -0,0 +1,66 @@
+type User {
+    id: ID!
+    displayName: String
+    age: Int
+    height: Int
+    sex: Sex
+    weight: Int
+    calorieGoal: Int
+    exerciseRecords(startTime: String, endTime: String): [ExerciseRecord!]!
+    foodRecords(startTime: String, endTime: String): [FoodRecord!]!
+}
+
+type CalorieGoal {
+  calories: Int!
+}
+
+type ExerciseRecord {
+    user: User!
+    calories: Int!
+    label: String!
+    timestamp: String!
+}
+
+type FoodRecord {
+    user: User!
+    calories: Int!
+    label: String!
+    timestamp: String!
+}
+
+enum Sex {
```

Comment:
might want to add "other" here to cover our bases

Reply:
This field is already optional on the user type, but we can add something to cover cases where a user has explicitly declined to specify a binary option for sex.